apps/condo/domains/miniapp/schema/B2BApp.js

```diff
@@ -48,6 +50,12 @@ const B2BApp = new GQLListSchema('B2BApp', {
         connectedMessage: CONNECTED_MESSAGE_FIELD,
         appUrl: IFRAME_URL_FIELD,
         isHidden: IS_HIDDEN_FIELD,
+        isGlobal: {
+            schemaDoc: 'Indicates whether the app is global or not. If so, then the application will be opened in hidden mode and receive various notifications from the condo. It\'s also possible to trigger some condo IFrame methods via global app outside of miniapps CRM section',
+            type: Checkbox,
+            defaultValue: false,
+            isRequired: true,
+        },
```

Comment:
May be add hook for check if `isHidden` has `!isGlobal` value?

Reply:
`isHidden` should always works for global B2B app. Since we need to turn-off any broken apps in runtime